app/src/main/java/com/safetyheads/akademiaandroida/launchScreen/LaunchScreenFragment.kt

```diff
@@ -0,0 +1,37 @@
+package com.safetyheads.akademiaandroida.launchScreen
+import android.os.Bundle
+import android.text.Spannable
+import android.text.SpannableString
+import android.text.style.ForegroundColorSpan
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.core.content.ContextCompat
+import androidx.fragment.app.Fragment
+import com.safetyheads.akademiaandroida.R
+import com.safetyheads.akademiaandroida.Footer
+import com.safetyheads.akademiaandroida.databinding.FragmentLaunchScreenBinding
+class LaunchScreenFragment : Fragment() {
+    private lateinit var binding: FragmentLaunchScreenBinding
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
```

Comment:
Please try limited other things rather than inflate and bind controls in this method from Fragment lifecycle.
If you what use view please do it in `onViewCreated()` method.

Same think you can find in Fragment.java:
```
It is recommended to only inflate the layout in this method and move logic that operates on the returned View to onViewCreated(View, Bundle).
If you return a View from here, you will later be called in onDestroyView when the view is being released.```